website/cue/reference/components/transforms/remap.cue

```diff
@@ -103,6 +103,15 @@ components: transforms: "remap": {
 		}
 	}
 
+	example_config: {
+		common: {
+			source: _example_vrl_program
+			advanced: {
+				drop_on_abort: true
+			}
```

Comment:
Unless the example page you posted is outdated (https://deploy-preview-9870--vector-project.netlify.app/docs/reference/configuration/transforms/remap/), it appears that this isn't being rendered correctly.

Reply:
Good catch